client/components/data/query-purchase-cancellation-offers/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCancellationOffers } from 'calypso/state/cancellation-offers/actions';
+import isFetchingCancellationOffers from 'calypso/state/cancellation-offers/selectors/is-fetching-cancellation-offers';
+
+interface OwnProps {
+	siteId?: number;
+	purchaseId?: number;
+}
+
+const QueryPurchaseCancellationOffers: React.FC< OwnProps > = ( { siteId, purchaseId } ) => {
```

Comment:
React.FC type already includes `children` property. If you do not want to accept `children` in this component, I would recommend typing it as below:

```suggestion
const QueryPurchaseCancellationOffers = ( { siteId, purchaseId }: OwnProps ) => {
```

Reply:
Nice tip, thanks!

Reply:
> React.FC type already includes `children` property.

That has changed since React 18, though we are still on v17.